data/lm/README.rst

```diff
@@ -8,18 +8,27 @@ lm.binary was generated from the LibriSpeech normalized LM training text, availa
    import os
```

Comment:
I know this is no part of your commit, but I'd always thought it odd that this is in the format of a Jupyter notebook.  Do you think it should stay a notebook or be migrated to something more coherent with the rest of the repo?